### ISSUE TITLE
Possible duplicated index

### ISSUE THREAD

**Issue by wallneradam** [tortoise/tortoise-orm#264]
**Describe the bug**
Not an important bug, maybe not a bug at all...
If I (mistakenly) use both `indexes` property of `class Meta` and index argument of a `Field` it creates 2 `KEY` rows (MySQL), with same id.

**To Reproduce**
```python
class Log(models.Model):
    class Meta:
        indexes = ('message_type',)
    message_type = fields.IntField(index=True)
```

**Expected behavior**
Index names should be unique. Also it could drop a warning in cases like this.

**Additional context**
This error was hidden until the last update, maybe because the MySQL dialect is now better.

**Comment by grigi** [comment]
Well, it's both a used wrong way bug, and a we could handle it better bug. 

One of the field refactoring goals is to build the ddl from the describe model func.
And there are many little issues with it that I feel like doing a big refactor on that to solve all our requirements.
But time is an issue.

**Comment by grigi** [comment]
Released v0.15.6 with index de-duplication. Please test :-)

**Comment by grigi** [comment]
Based on my testing, it does the right thing. So I'll close this issue.
